### ISSUE TITLE
Add ability to send/encrypt payload in PTM via Quorum

### ISSUE THREAD

**Issue by prd-fox** [Consensys-Incorporated/quorum#826]
### Summary

Currently we have a way to retrieve a payload from the Private Transaction Manager (`eth.getQuorumPayload`), but it might be nice to have one that sends a payload to the PTM (both Tessera raw txns, and regular ones).

This has a few upsides:
- the security for sending transactions to the PTM (for raw txns) is purely via the unix socket and does not need to be exposed to another network interface.
- the same security used for protecting Quorum APIs can be used

---

This needs some discussion first as to whether this is actually something we want to add, and if so, what exactly is required.

### Acceptance Criteria

1. Dedicated RPC method (name TBD) for calling `send` on the Private Transaction Manager, which specifies the private payload, the private Private Tx Manager (PTM) public key and the recipient PTM public keys. The result (a 64 byte hash) should be able to be used by all the of involved nodes to retrieve said payload with `eth.getQuorumPayload`.

2. Dedicated RPC method (name TBD) for calling `storeraw` , which specifies the private payload and the private Private Tx Manager (PTM) public key. The result (a 64 byte hash) should be able to be put into a private transaction and submitted using the `eth_sendPrivateRawTransaction` RPC method.

**Comment by ricardolyn** [comment]
@ricardolyn describe AC.

create a new JSON RPC route that does the storeraw to tessera instead having to be done by client.

**Comment by ricardolyn** [comment]
@prd-fox is there any Acceptance Criteria that you think should be added or any description more that should be mentioned on this ticket? I'm trying to refine to work on it in the future.

**Comment by prd-fox** [comment]
@ricardolyn I've added some extra info in the acceptance criteria, _if_ this is a feature we want to implement. However, it may not be something worth adding:

- For scenario 1 (calling `send`), there is no advantage for a user to do this, since they are doing on-node signing and that is already handled during the process. This would only be useful if they want to use Tessera as a communication tool, i.e. distributing payloads securely without putting them in transactions.

- For scenario 2, `eth_fillTransaction` now exists since version 2.6.0 which covers this scenario. It will call `storeraw` on the Private Tx Manager and return the altered transaction for signing.

I added this ticket for discussion on whether this feature is a useful idea, and now I am leaning towards no, but interested to hear your thoughts.
